eva-accession-remapping/src/main/java/uk/ac/ebi/eva/accession/remapping/batch/processors/SubmittedVariantToVariantContextProcessor.java

```diff
@@ -78,19 +76,20 @@ public VariantContext process(SubmittedVariantEntity variant) {
         if (variant.getClusteredVariantAccession() != null) {
             attributes.put(RS_KEY, RS_PREFIX + variant.getClusteredVariantAccession());
         }
+        attributes.put(PROJECT_KEY, replaceInvalidCharacters(variant.getProjectAccession()));
         return attributes;
     }
 
     /**
      * In VCF, in the INFO column, keys and values can not have spaces, commas, semicolons or equal signs. Specially
      * the study IDs from dbSNP are likely to contain some of those letters.
      *
-     * TODO jmmut: use percentage encoding?
+     * See section 1.2 of VCFv4.3 (https://samtools.github.io/hts-specs/VCFv4.3.pdf)
      */
-    private List<String> replaceInvalidCharacters(List<String> infoValues) {
-        return infoValues.stream()
-                         .map(s -> s.replaceAll("[ ,;=]", "_"))
-                         .collect(Collectors.toList());
+    private String replaceInvalidCharacters(String infoValues) {
+        return infoValues.replaceAll("[,]", "%2C")
+                         .replaceAll("[;]", "%3B")
+                         .replaceAll("[=]", "%3D");
```

Comment:
I am guessing % and space don't need to be here because they wouldn't pass the validation stage?

Reply:
I'm going to add the entry for %, but spaces are allowed in INFO values.